### DISCUSSION TITLE
New AMD driver version is available notification.

### DISCUSSION THREAD

**Discussion by fabcard** [GSDragoon/RadeonSoftwareSlimmer#51] [General]
Hello @GSDragoon 
I'm new using this tool, and I'm following this guide from AtlasOS https://docs.atlasos.net/getting-started/post-installation/drivers/gpu/amd/
I would like to know what is the system service or scheduled task responsible for informing me when a new AMD driver version is available. Please let me know. What settings must I enable or disable in order to receive a new driver available notification?
Furthermore, it would be helpful to have this information visible on the tool.
Thanks in advance.

**Comment by GSDragoon** [comment]
I'm not actually sure, it's not something I personally use. My guess would be the main Radeon Software Settings process which will startup by default.

**Reply by fabcard** [reply]
Oh. I've got the check for updates enabled in the Adrenalin settings, but I never get notified when a new driver is released.
I appreciate your assistance and this fantastic tool.
